javascript-data-model/order-history.js

```diff
@@ -0,0 +1,40 @@
+var orderHistory = [
+  {
+    'order #': '114-3941689-8772232',
+    'order placed': '08/4/2020',
+    'total items': 1,
+    products: 'JavaScript for impatient programmers',
+    total: '$34.00',
+    delivered: '08/8/2020',
+    'ship to': 'JS Masher'
+  },
+  {
+    'order #': '113-9984268-1280257',
+    'order placed': '07/19/2020',
+    'total items': 1,
+    products: 'The Timeless Way of Building',
+    total: '$44.53',
+    delivered: '07/20/2020',
+    'ship to': 'JS Masher'
+  },
+  {
+    'order #': '114-2875557-9059409',
+    'order placed': '07/4/2020',
+    'total items': 1,
+    products: 'Gamecube Controller Adapter',
+    total: '$17.22',
+    delivered: '07/7/2020',
+    'ship to': 'JS Masher'
+  },
+  {
+    'order #': '113-2883177-2648248',
+    'order placed': '07/3/2020',
+    'total items': 2,
+    products: ['GameCube Controller', 'The Art of Sql'],
+    total: '$138.93',
+    delivered: '07/5/2020',
+    'ship to': 'JS Masher'
```

Comment:
How would you model the data with each product price? We only have title for each product but without prices / author / return window date

Reply:
Also, I understand you've used 'property name with space` for properties here and there since it's still a valid property name. There is nothing wrong with it but usually when we write Javascript, we use camelCase as a basic naming convention: https://www.w3schools.com/js/js_conventions.asp (it's a best practice)